docs/runbooks/chart-dependency-update-process.md

```diff
@@ -0,0 +1,127 @@
+# Chart Dependencies Update Process
+
+This document outlines the process for upgrading chart dependencies in the Chainloop project. The current dependencies are Vault and PostgreSQL, both based on Bitnami Charts, with vendorized versions stored in the repository.
+
+## Overview
+
+The Chainloop project uses vendorized Helm charts for its dependencies:
+- **PostgreSQL**: Database backend (Bitnami chart)
+- **Vault**: Secret storage (Bitnami chart)  
+- **Dex**: OIDC provider (self-managed, not upgraded via this process)
+
+## Step-by-Step Upgrade Process
+
+### 1. Check Container Image Versions
+
+1. Visit [Bitnami Containers](https://github.com/bitnami/containers)
+2. Find the specific image (e.g., `bitnami/postgresql` or `bitnami/vault`)
+3. Navigate the folder structure:
+    - `2/debian-12` means:
+        - `2`: major branch version
+        - `debian-12`: distribution version
+4. In the commit history, look for "Release ..." commits to find the container image version
+5. Inside the `Dockerfile`, check the `APP_VERSION` environment variable
+    - This is the application version used in the vendorized chart
+
+### 2. Locate the Chart to Upgrade
```

Comment:
shouldn't we do first an update of the Chart, from there find the associated images?